### PR TITLE
update-query-tutorial

### PR DESCRIPTION
**Pull request by architgoyal1**
- Add Preset example

### CODE REVIEW

File: docs/try-snowplow/accessing-and-querying-your-try-snowplow-data/index.md [hunk 1]
```diff
@@ -6,12 +6,72 @@ sidebar_position: 30
...
+![](images/5-finalizesetup.png)
+
+6. On the top bar, select the “SQL” section and “SQL lab” subsection.
+
+<img src=images/6-sqleditor.png width="400"/>\
+
+
+7. Copy SQL query from the [Try Snowplow UI](https://try.snowplowanalytics.com/access-data)
+
+```
```
Comment (stanch) [inline docs/try-snowplow/accessing-and-querying-your-try-snowplow-data/index.md:57]:
````suggestion
```sql
````

Comment (stanch) [reply docs/try-snowplow/accessing-and-querying-your-try-snowplow-data/index.md:57]: (Makes it nicely highlighted)

File: docs/try-snowplow/accessing-and-querying-your-try-snowplow-data/index.md [hunk 2]
```diff
@@ -6,12 +6,72 @@ sidebar_position: 30
...
+
+![](images/7-runquery.png)
+
+You should now see Snowplow atomic data which you can [explore](https://docs.snowplow.io/docs/understanding-your-pipeline/canonical-event/).
+
+
+### Example: Create a simple time-series chart
+From the query results, select “Create chart” and select “Bar chart”, set these values and click “Update chart”:
+
+* X-axis: derived_tstamp
```
Comment (stanch) [inline docs/try-snowplow/accessing-and-querying-your-try-snowplow-data/index.md:72]:
```suggestion
* X-axis: `derived_tstamp`
```

File: docs/try-snowplow/accessing-and-querying-your-try-snowplow-data/index.md [hunk 3]
```diff
@@ -6,12 +6,72 @@ sidebar_position: 30
...
+* Metrics:
+  * Column: “event_id”
```
Comment (stanch) [inline docs/try-snowplow/accessing-and-querying-your-try-snowplow-data/index.md:74]:
```suggestion
  * Column: `event_id`
```

File: docs/try-snowplow/accessing-and-querying-your-try-snowplow-data/index.md [hunk 4]
```diff
@@ -6,12 +6,72 @@ sidebar_position: 30
...
+  * Aggregate: COUNT
```
Comment (stanch) [inline docs/try-snowplow/accessing-and-querying-your-try-snowplow-data/index.md:75]:
```suggestion
  * Aggregate: `COUNT`
```

File: docs/try-snowplow/accessing-and-querying-your-try-snowplow-data/index.md [hunk 5]
```diff
@@ -6,12 +6,72 @@ sidebar_position: 30
...
+
+![](images/2-presetsetup.png)
+
+3. Select PostgreSQL as your database
+
+![](images/3-selectpostgres.png)
+
+4. Enter the connection details provided in the [Try Snowplow UI](https://try.snowplowanalytics.com/access-data): Hostname, Port, Database, Username and Password (request credentials on Try UI if you haven’t done so)
+
+<img src=images/4-connectpostgres.png width="400"/>\
```
Comment (stanch) [inline docs/try-snowplow/accessing-and-querying-your-try-snowplow-data/index.md:44]: Resize the image itself and use a markdown image syntax (as discussed on slack)

Comment (architgoyal1) [reply docs/try-snowplow/accessing-and-querying-your-try-snowplow-data/index.md]: Thanks Nick, updated

**Review by stanch — APPROVED** [review]

File: docs/try-snowplow/accessing-and-querying-your-try-snowplow-data/index.md [hunk 6]
```diff
@@ -6,12 +6,71 @@ sidebar_position: 30
...
+
+4. Enter the connection details provided in the [Try Snowplow UI](https://try.snowplowanalytics.com/access-data): Hostname, Port, Database, Username and Password (request credentials on Try UI if you haven’t done so)
+
+![](images/4-connectpostgres.png)
+
+5. Continue with default settings and finalize set up
+
+![](images/5-finalizesetup.png)
+
+6. On the top bar, select the “SQL” section and “SQL lab” subsection.
```
Comment (stanch) [inline docs/try-snowplow/accessing-and-querying-your-try-snowplow-data/index.md:50]:
```suggestion
6. On the top bar, select the “SQL” section and “SQL lab” subsection
```

**Pull request merged**